packages/writing-style/bin/commercetools-vale.js

```diff
@@ -8,7 +8,16 @@ const binaryFileName = os.platform() === 'win32' ? 'vale.exe' : 'vale';
 
 const valeBinary = path.join(__dirname, `./${binaryFileName}`);
 const valeConfig = path.join(__dirname, `../.vale.ini`);
-const result = shelljs.exec(
-  `${valeBinary} --config ${valeConfig} ${process.argv.slice(2).join(' ')}`
-);
+let params = process.argv.slice(2);
+// simplistic check whether the --config param was passed empty by the VSCode plugin (empty = followed by another flag)
+const configParamPosition = params.findIndex((p) => p === '--config');
+if (
+  params[configParamPosition + 1] &&
+  params[configParamPosition + 1].startsWith('-')
```

Comment:
I don't fully understand what this check is for, if you then just do a filter.

Reply:
this is checking whether the parameter _after_ the `--config` is not a value but another config flag like `-v` or `--format` etc. .    The config should be filtered out only if it has no value (you seem to be able to pass it twice the program takes the last one, so it's OK to pass a config to our wrapper to override our config.

Reply:
Ah ok, got it now